src/styles/_variables.scss

```diff
@@ -1,25 +1,51 @@
-// Box Design Language Theme Colors
+// Box Design Language Colors, confirm with design before extending
+// @see https://gallery.io/projects/MCHbtQVoQ2HCZTdZ0F79-fQJ/files/MCEJu8Y2hyDScSTYfw7hxWcA-L66cSQvSj4
+
+// Box/Primary Blues
 $bdl-box-blue: #0061d5 !default;
-$bdl-secondary-blue: #003c84 !default;
+$bdl-box-blue-50: #619de5 !default;
+$bdl-box-blue-05: #f2f7fd !default;
+
+// Box/Primary Grays
+$bdl-gray: #222 !default;
+$bdl-gray-62: #767676 !default;
+$bdl-gray-50: #909090 !default;
+$bdl-gray-20: #d3d3d3 !default;
+$bdl-gray-10: #e8e8e8 !default;
+$bdl-gray-05: #f4f4f4 !default;
+$bdl-gray-02: #fbfbfb !default;
+
+// Dark Blues
+$bdl-dark-blue: #003c84 !default;
+
+// Light Blues
 $bdl-light-blue: #2486fc !default;
-$bdl-green-light: #26c281 !default;
+
+// Yelloranges
 $bdl-yellorange: #f5b31b !default;
+$bdl-yellorange-10: #fef7e8 !default;
+
+// Yellows
 $bdl-yellow: #ffd700 !default;
+$bdl-yellow-10: #fffbe5 !default;
+
+// Green Lights
+$bdl-green-light: #26c281 !default;
+
+// Grimaces
 $bdl-grimace: #4826c2 !default;
+
+// Purple Rains
 $bdl-purple-rain: #9f3fed !default;
-$bdl-watermelon-red: #ed3757 !default;
 
-// Box Design Theme Color Variations
-// Confirm with design before extending
-$bdl-yellorange-10: #fef7e8 !default;
-$bdl-yellow-10: #fffbe5 !default;
-$bdl-box-blue-50: #619de5 !default;
-$bdl-box-blue-05: #f2f7fd !default;
+// Watermelon Reds
+$bdl-watermelon-red: #ed3757 !default;
 
-// Box Design Language Neutral Colors
-$bdl-neutral-01: #222 !default;
-$bdl-neutral-02: #767676 !default;
-$bdl-neutral-03: #909090 !default;
+// Do not use, deprecated colors below this line
+$bdl-secondary-blue: $bdl-dark-blue !default;
+$bdl-neutral-01: $bdl-gray !default;
+$bdl-neutral-02: $bdl-gray-62 !default;
+$bdl-neutral-03: $bdl-gray-50 !default;
 $bdl-neutral-04: #eaeaea !default;
```

Comment:
these should be remapped as well, the 100-point scale is an evolution of this and they should have the same source of truth

Reply:
Let me see if I can map it to the closest grays, hmm